### DISCUSSION TITLE
CsvConfiguration - Parameterless Constructor

### DISCUSSION THREAD

**Discussion by TonyValenti** [JoshClose/CsvHelper#1666] [Ideas]
@JoshClose -
Can you update CsvConfiguration to have a parameterless constructor that defaults the culture to ```CultureInfo.InvariantCulture```?

**Comment by JoshClose** [comment]
No because there were too many "bugs" where people were just using the wrong culture. Several every week. It's meant to make the user think about what they're doing. Now that I think about it; I don't think I've seen a single one come across since requiring it.
